interface/modules/zend_modules/module/Carecoordination/src/Carecoordination/Model/CcdaServiceDocumentRequestor.php

```diff
@@ -49,7 +49,7 @@ public function socket_get($data)
                         error_log("Failed to close pipehandle for ccdaservice");
                     }
                 } else {
-                    $command = 'nodejs';
+                    $command = 'node';
```

Comment:
Are we sure that people installing on linux with apt that the apt package doesn't still assign node to nodejs alias?
Newer snap doesn't. Making 'node' the primary check is fine but to be safe we still need to check for nodejs. For windows this isn't an issue as it always used 'node'.